modules/components/AsyncConnect.js

```diff
@@ -44,7 +44,9 @@ export default class AsyncConnect extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    this.loadAsyncData(nextProps);
+    if (!this.context.store.getState().reduxAsyncConnect.loading) {
```

Comment:
`componentDidMount` and `componentWillReceiveProps` fire in succession on initial load, don't load if we're already loading.